Packs/GoogleDrive/Integrations/GoogleDrive/README.md

```diff
@@ -1304,3 +1304,52 @@ Modify labels to file.
 | GoogleDrive.Labels.labels.id | String | The label id of the label to set | 
 | GoogleDrive.Labels.labels.kind | String | The type of resource. This is always drive\#label | 
 | GoogleDrive.Labels.labels.revisionId | String | The revision id of the label | 
+
+### google-drive-file-copy
+
+***
+Make a copy of a Google Drive file.
+
+#### Base Command
+
+`google-drive-file-copy`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| file_id | The ID of the file to copy. | Required | 
+| copy_title | The name of the copied file. | Required | 
+| user_id | The user's primary email address. | Optional | 
+| supports_all_drives | Whether the requesting application supports both My Drives and shared drives. Possible values are: true, false. Default is false. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| GoogleDrive.kind | String | The content type of the file. | 
+| GoogleDrive.id | String | The ID of the copied file. | 
+| GoogleDrive.name | String | The name of the copied file. | 
+| GoogleDrive.mimeType | String | The MIME type of the copied file. | 
+
+#### Command example
+```!google-drive-file-copy file_id="1O8Gx7DslVpbd-HN7lp4MIN1DDakpw-bHVHCwir2wUlo" copy_title="New Copy"```
+#### Context Example
+```json
+{
+    "GoogleDrive": {
+        "id": "1h1SfvpTwAYPnKB7oaf2lP3ztjgc-Wz2FeI1j14zwk3w",
+        "kind": "drive#file",
+        "mimeType": "application/vnd.google-apps.spreadsheet",
+        "name": "New Copy"
+    }
+}
+```
+
+#### Human Readable Output
+
+>### New file copied from *1O8Gx7DslVpbd-HN7lp4MIN1DDakpw-bHVHCwir2wUlo*
+>|Id|
```

Comment:
why only have id in readable output?

Reply:
Keeping it consistent with the other googledrive functions.

Reply:
I added more info